apps/testingaccessibility/src/pages/api/sanity/webhooks/add-records-to-algolia.ts

```diff
@@ -1,14 +1,11 @@
 import {NextApiRequest, NextApiResponse} from 'next'
-import type {SanityClient} from '@sanity/client'
 import {sanityClient} from 'utils/sanity-client'
 import {sanityAlgolia} from 'utils/algolia'
 import {setupHttpTracing} from '@vercel/tracing-js'
 import {withSentry} from '@sentry/nextjs'
 import {tracer} from 'utils/honeycomb-tracer'
 import {isValidRequest} from '@sanity/webhook'
 
-const secret = process.env.NEXT_PUBLIC_ALGOLIA_API_WRITE_KEY as string
```

Comment:
this was definitely an error since the Algolia write key would be very unlikely to also be the Sanity webhook signing secret (I checked, and they weren't the same)